content/_data/authors/jglick.adoc

```diff
@@ -0,0 +1,8 @@
+---
+name: "Jesse Glick"
+twitter: tyvole
+github: jglick
+---
+
+Jesse has been developing Jenkins core and plugins for years.
+He is the coauthor with Kohsuke of the Pipeline system.
```

Comment:
Maybe needs some adjustment since the ecosystem has dozens of contributors now.
Maybe something like Pipeline core.